test/e2e/antreapolicy_test.go

```diff
@@ -4409,14 +4411,48 @@ func TestAntreaPolicy(t *testing.T) {
 
 	t.Run("TestMulticastNP", func(t *testing.T) {
 		skipIfMulticastDisabled(t, data)
-		t.Run("Case=MulticastNPIGMPQueryAllow", func(t *testing.T) { testACNPIGMPQueryAllow(t, data) })
-		t.Run("Case=MulticastNPIGMPQueryDrop", func(t *testing.T) { testACNPIGMPQueryDrop(t, data) })
-		t.Run("Case=MulticastNPPolicyEgressAllow", func(t *testing.T) { testACNPMulticastEgressAllow(t, data) })
-		t.Run("Case=MulticastNPPolicyEgressDrop", func(t *testing.T) { testACNPMulticastEgressDrop(t, data) })
+		t.Run("noEncap", func(t *testing.T) {
```

Comment:
Why run testMulticastNP only with "noEncap" mode? In fact, I didn't find where you check or update configurations to ensure the cluster is noEncap. Is the case name correct?

Reply:
updated